org.jellyfin.JellyfinServer.yml

```diff
@@ -1,16 +1,17 @@
 id: org.jellyfin.JellyfinServer
 runtime: org.freedesktop.Platform
-runtime-version: "22.08"
+runtime-version: '23.08'
```

Comment:
24.08 is the current, maybe jumping to it now it the best course.

Reply:
24.08 did not exist back then. It was my plan to update the runtime 24.08 for v10.10 of the application, which I did. https://github.com/flathub/org.jellyfin.JellyfinServer/pull/432